src/applications/debt-letters/tests/DebtCardsList.unit.spec.js

```diff
@@ -169,7 +169,12 @@ describe('DebtLettersSummary', () => {
   });
   it('renders correct number of debt cards', () => {
     const wrapper = shallow(<DebtCardsList store={fakeStore} />);
-    expect(wrapper.dive().find(`Connect(DebtLetterCard)`).length).to.equal(4);
+    expect(
+      wrapper
+        .dive()
```

Comment:
`Provider` seems to have an extra component layer now, I believe due to `React.memo`.